parser_library/src/context/hlasm_context.cpp

```diff
@@ -336,7 +336,7 @@ void hlasm_context::push_statement_processing(const processing::processing_kind
 
 void hlasm_context::pop_statement_processing()
 {
-    if (proc_stack_.back().owns_source)
+    if (!proc_stack_.empty() && proc_stack_.back().owns_source)
         source_stack_.pop_back();
 
     proc_stack_.pop_back();
```

Comment:
If `proc_stack_` is `empty` then `pop_back` makes no sense. Also I was under the impression that `proc_stack_` should never be empty.

Reply:
Yes, it makes no sense when it is empty. I was getting this vector empty when calling end_processing() directly when there are two end instruction. But later used finished_flag and forgot to remove this check. Sorry!